clang/lib/Parse/ParseOpenMP.cpp

```diff
@@ -2314,10 +2314,12 @@ Parser::DeclGroupPtrTy Parser::ParseOpenMPDeclarativeDirectiveWithExtDecl(
       // Here we expect to see some function declaration.
       if (AS == AS_none) {
         assert(TagType == DeclSpec::TST_unspecified);
-        ParsedAttributes EmptyDeclSpecAttrs(AttrFactory);
-        MaybeParseCXX11Attributes(Attrs);
+        ParsedAttributes DeclSpecAttrs(AttrFactory);
```

Comment:
Why isn't this calling the new function?  It seems you've added the  `ParsingDeclSpec` default argument correctly, so I don't see a reason to not have this call the new one too.

Reply:
Because `Attrs` is not a local variable but an argument passed to `ParseOpenMPDeclarativeDirectiveWithExtDecl`